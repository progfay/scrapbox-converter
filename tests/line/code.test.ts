/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('code', () => {
  it('Simple code', () => {
    const input = `Code
\`code\``
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
