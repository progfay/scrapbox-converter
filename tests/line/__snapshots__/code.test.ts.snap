// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`code Simple code 1`] = `
"<article>
  <h1 class=\\"title\\">Code</h1>
  <section>
    <p><pre>code</pre>
    </p>
  </section>
</article>"
`;
